test/unit/controller/transaction-controller.ts

```diff
@@ -579,12 +579,13 @@ describe('TransactionController', (): void => {
         .get(`/transactions/${trans.id}`)
         .set('Authorization', `Bearer ${ctx.organMemberToken}`);
       expect(res.status).to.equal(200);
-      expect(ctx.specification.validateModel(
+      // TODO Fix the BasePointOfSaleResponse; it always contains a revision and useAuth but this is not reflected in the swagger.
```

Comment:
What is this TODO?

Reply:
The `product` in the `SubTransactionRowResponse` is a `BaseProductResponse` but also has a `revision` so its neither a `BaseProductResponse` nor a `ProductResponse`.

See line 387 of the `transaction-service.ts`

Reply:
If you open an issue for this (i.e. removing revision from the responses to make it transparent for endusers), I am fine with this. Please refer to the issue on this line.